website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -31,31 +37,60 @@ you'll need to make sure the following outbound traffic is allowed:
 | N/A              | Varies               | `3478`          | STUN            | STUN protocol signaling    |
 | N/A              | Varies               | `49152 - 65535` | TURN            | TURN protocol channel data |
 
-## Resource considerations
+### Where to deploy Gateways
+
+Ideally, Gateways should be deployed as close to the resources they're serving
+-- in some cases, even on the same physical host. This ensures the lowest
+possible latency and highest possible throughput for client connections, and
+allows you to more easily deploy additional gateways for a particular resource
+that needs to handle more client connections.
+
+The downside of deploying gateways close to resources is that it increases the
+number of overall gateways you need to deploy, possibly making it more difficult
+to manage and monitor them. Firezone's control plane is designed to make this as
+easy as possible, but it's still something to keep in mind when planning your
+deployment.
```

Comment:
Maybe we should push more for zero trust here rather than convincing that it "will be hard"? I would even reference a few standards/good sources explaining why zero trust is important.

Reply:
Updated to say

> The downside of deploying gateways close to resources is that it increases the
number of overall gateways you need to manage and monitor. Firezone was
carefully designed however to minimize this downside, making it easy to manage
even hundreds of gateways from the admin portal.